transformations/p1_noun_transformation/sample.py

```diff
@@ -0,0 +1,5 @@
+from transformations.p1_noun_transformation import AddNounDefinition
```

Comment:
Instead, better to move this sample code in AddNounDefinition and comment that out (as it's just for testing). You can follow buttern_finger_transformation.

Reply:
done